CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 # Upcoming
 
 ### ✅ Added
+- Add support for `enforce_unique` parameter on sending reactions
```

Comment:
This is an LLC change, so you need to use the changelog located in `Sources/StreamChat`

Reply:
.... and we have just one CHANGELOG again, so ignore this :)